apps/web/app/environments/[environmentId]/surveys/[surveyId]/edit/SurveyMenuBar.tsx

```diff
@@ -39,6 +44,20 @@ export default function SurveyMenuBar({
     }
   }, [activeId, audiencePrompt]);
 
+  useEffect(() => {
+    const warningText = "You have unsaved changes - are you sure you wish to leave this page?";
```

Comment:
<img width="410" alt="Screenshot 2023-06-21 at 14 28 56" src="https://github.com/formbricks/formbricks/assets/675065/4ab44630-3a72-48ef-a6d5-0da9e378e2a5">
 
I always get this default text that seems to come from the browser (chromium). When is your `warningText` used?

Reply:
This is not supported in most modern browsers. but we have to set at least an empty string. So I thought it's better to display the message in supported browsers. 🙂

https://stackoverflow.com/a/45088935